src/apiserver/api_server_impl.cc

```diff
@@ -428,7 +428,25 @@ void APIServerImpl::RegisterGetTable() {
                           writer << err.Set("Invalid path");
                           return;
                       }
+                      std::vector<std::string> dbs;
+                      hybridse::sdk::Status status;
+                      auto ok = sql_router_->ShowDB(&dbs, &status);
+                      if (!ok) {
+                          writer << err.Set(status.msg);
+                          return;
+                      }
                       auto db = db_it->second;
+                      bool db_ok = std::find(dbs.begin(), dbs.end(), db) != dbs.end();
+                      if (!db_ok) {
+                          writer.StartObject();
+                          writer.Member("code") & -1;
+                          writer.Member("msg") & std::string("DB not found");
+                          writer.Member("tables");
```

Comment:
If failed, do we need the 'tables' field? code & msg is enough?

Reply:
In my opinion, whether to keep the 'tables' field are both ok. Here I choose to keep it because: First, the additional content is only _"tables": []_. It is very short. Second, in my experience using MVVM front-end frameworks such as vue.js, it is better to keep it. Because the code would also work even if the developer doesn't do a special check. Nor the variable binding to the component will be an 'undefined'.

Reply:
But we’ve defined the error response format(code&msg only). If you think we need to keep the empty field, it should be consistent in all methods. Need more discussion.

Reply:
Yes if other methods only have code & msg, deleting it is better.